src/generate-banner.js

```diff
@@ -0,0 +1,31 @@
+define(['lodash', 'package'], function(_, pkg) {
+
+  // Return a Modernizr file banner.
+  // Usage:
+  //   require('banners')(type);
+  // Arguments:
+  //   type (String, required): either 'compact' or 'full'.
+
+  return function banners(type) {
+    if (!type || type === 'compact') {
+      return '/*! ' + pkg.name + ' ' + pkg.version + ' (Custom Build) | ' + pkg.license  + ' */\n';
+    }
+    else if (type === 'full') {
+      return '/*!\n' +
+        ' * ' + pkg.name + ' v' + pkg.version + '\n' +
+        ' * modernizr.com\n' +
+        ' *\n' +
+        ' * Copyright (c)\n *  ' + _.pluck(pkg.contributors, 'name').join('\n *  ') + '\n\n' +
```

Comment:
eh - its only in the debug build. I'm fine with having eternal glory on poorly optimized sites.